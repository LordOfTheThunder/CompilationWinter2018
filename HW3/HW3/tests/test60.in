void CheckBinary() {
	int x = 3 - 2;
	byte y = 5b / 3b;
	int z = 3 + 3b;
}

void main() { print("Max is cool"); }
