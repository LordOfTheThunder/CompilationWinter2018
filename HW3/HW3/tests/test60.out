---end scope---
x INT 0
y BYTE 1
z INT 2
---end scope---
print (STRING)->VOID 1
printi (INT)->VOID 2
CheckBinary ()->VOID 3
